test/access-token.test.js

```diff
@@ -289,6 +289,33 @@ describe('loopback.token(options)', function() {
         });
     });
 
+  it('should generate a 401 on a current user literal route without an authToken',
+    function(done) {
+      var app = createTestApp(null, done);
+      request(app)
+        .get('/users/me')
+        .set('authorization', null)
+        .expect(401)
+        .end(function(err, res) {
+          assert(!err);
+          done();
+        });
+    });
+
+  it('should generate a 401 on a current user literal route with invalid authToken',
+    function(done) {
+      var app = createTestApp(this.token, done);
+      var invalidTokenStub = {id: 'stub id'};
+      request(app)
+        .get('/users/me')
+        .set('Authorization', invalidTokenStub.id)
```

Comment:
No need to create `invalidTokenStub` object, just pass a string that makes it clear it's an invalid token id.

```js
.set('Authorization', 'invalid-token-id')
```